site/src/load.rs

```diff
@@ -64,23 +64,32 @@ impl TryCommit {
     }
 }
 
+/// Keys for accessing various services
+///
+/// At the moment only used for accessing GitHub
 #[derive(Debug, Default, Deserialize)]
 pub struct Keys {
+    /// GitHub API token from the `GITHUB_API_TOKEN` env variable
     pub github: Option<String>,
+    /// GitHub webhook secret from the `GITHUB_WEBHOOK_SECRET` env variable
     pub secret: Option<String>,
 }
 
+/// Site configuration
 #[derive(Debug, Deserialize)]
 pub struct Config {
     pub keys: Keys,
 }
 
+/// Site context object that contains global data
 pub struct InputData {
```

Comment:
We might want to consider renaming this. InputData is descriptive enough to make me think it's something specific but not descriptive enough to know what that might be. 

Considering it's sort of a collection of cached items we just want to keep around, something purposefully vague like `SiteCtx` might be better.

Reply:
Happy to see this renamed -- I didn't ever come up with a nice name for it. I think SiteCtx is not a bad idea.